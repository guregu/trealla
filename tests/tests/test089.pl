main :-
	freeze(X,writeln(here)), X \= true.

:- initialization(main).
