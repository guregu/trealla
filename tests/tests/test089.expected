here
Warning: call(main) failed
